Documentation/planning/arcade-powered-source-build/implementation-plan.md

```diff
@@ -37,21 +37,9 @@ To get each repo building with the new source-build 5.0 plan, [Arcade-Powered So
 | ❗ | At Risk |
 
 ## Stage descriptions:
-  - **(1) Build from Source 5.0** – Get repo building from source with 5.0 source in dotnet/source-build.
-  - **(2) Patch Removal** – Incorporate patches required to build from source into the repo code base.
-  - **(3) Move Build Params to Eng Dir** – Move all source-build specific build parameter from `/repos/<reponame>.proj` in source-build repo to repo `/eng` directory.
-  - **(4) Input Intermediate Packages Available** – Indicates upstream intermediate packages are available, and work on this repo can begin.
-  - **(5) Official build** – Build source-built intermediate packages in official build.
-  - **(6) Source-build PR validation** – Add AzDO build jobs to PR validation to ensure build-from-source continues running without avoidable regressions.
-  - **(7) Prebuilt check / enforcement** – Add a check to PR validation to ensure that no prebuilts are included when building repo.
-
-## Key Milestone Dates:
-
-| Stage(s) | Start | End |
-| --- | :---: | :---: |
-| Stage 1 - Build from Source 5.0 | 2020-05-28 | 2020-06-29 |
-| Stage 2 - Patch Removal | 2020-06-16 | 2020-07-23 |
-| Stages 3-5 - Get into Official Build | 2020-06-25 | 2020-09-16 |
-| Stages 6-7 - PR Validation / Prebuilt check | 2020-07-28 | 2020-10-05 |
-
-
+  - **(1) Build from source 5.0** – Get repo building from source with 5.0 source in dotnet/source-build.
+  - **(2) Input intermediate packages available** – Indicates upstream intermediate packages are available, and work on this repo can begin.
+  - **(3) Merge patches & local build infra into dev branch** – Repo owner has created source-build-dev branch.  Move all source-build specific build parameters from `/repos/<reponame>.proj` in source-build repo to repo `/eng` directory and add repo-specific patches into source-build-dev branch.
```

Comment:
Should we say `dev/source-build branch`? At least, that's what I expect the name to be. (Although it doesn't matter, if the repo owner disagrees that's fine.)

Otherwise, `source-build dev branch` would read better IMO.